add strings for translation

rglbr opened the pull request:
fixes https://github.com/ornicar/lila/issues/9546

kraktus commented:
Hey,

You don't need to convert these to `String`. Use `myTranslationKey()` instead of `myTranslationKey.txt()` when possible. Also `performance` has already a key in `site.xml` Always good to check when you see really short/possibly common translation 👍

rglbr commented:
> Hey,
> 
> You don't need to convert these to `String`. Use `myTranslationKey()` instead of `myTranslationKey.txt()` when possible. Also `performance` has already a key in `site.xml` Always good to check when you see really short/possibly common translation +1

Thank you for the review. I have addressed the comments

kraktus commented:
Refractored because keys should ideally be one complete sentence when possible. Some languages do not end their sentences with `.` for example.

ornicar commented on app/views/puzzle/dashboard.scala:
```diff
@@ -157,26 +157,26 @@ object dashboard {
         results.canReplay option span(cls := s"$metricClass--fix__text")(
           strong(results.unfixed),
-          span("to replay")
+          span(trans.puzzle.toReplay())
```
this should use a translation argument like https://github.com/ornicar/lila/blob/c2c39a5743ea9d96bfbdc939ebf792c768816292/translation/source/puzzle.xml#L32-L35

rglbr replied:
resolved in cf6a1d9ee65be1049b48e4cf0ea863e8f08d6e7a

ornicar commented on app/views/puzzle/dashboard.scala:
```diff
@@ -157,26 +157,26 @@ object dashboard {
       )(
         strong(s"${results.winPercent}%"),
-        span("solved")
+        span(trans.puzzle.solved())
```
this should use a translation argument like https://github.com/ornicar/lila/blob/c2c39a5743ea9d96bfbdc939ebf792c768816292/translation/source/puzzle.xml#L30

rglbr replied:
resolved in cf6a1d9ee65be1049b48e4cf0ea863e8f08d6e7a

ornicar commented on app/views/puzzle/dashboard.scala:
```diff
@@ -157,26 +157,26 @@ object dashboard {
       div(cls := s"$metricClass $metricClass--played")(
         strong(results.nb.localize),
-        span("played")
+        span(trans.puzzle.played())
```
this should use a translation argument like https://github.com/ornicar/lila/blob/c2c39a5743ea9d96bfbdc939ebf792c768816292/translation/source/puzzle.xml#L32-L35

rglbr replied:
do you recommend using plurals with translation arguments or translation arguments alone ? imo, the existing text(without plurals) looks good and we can add the arguments alone

benediktwerner replied:
Not sure what you mean? It looks good in English but this doesn't work in languages where "played" changes depending on the number of games or where the number of games will have to come after the word or in between something.

rglbr replied:
I feel that plurals are not needed for these tiles
![image](https://user-images.githubusercontent.com/12563895/129464182-110cc2fa-c2e4-4fb4-bdc5-a8d5e6b68fb1.png)

So my question is,
do we need to create something like

`xml
<plurals name="Xplayed"> 
   <item quantity="one">% played</item> 
   <item quantity="other">% played</item> 
 </plurals> 
`
or something like

`xml
<string name="Xplayed">%s played</string>
`
would suffice

rglbr replied:
resolved in cf6a1d9ee65be1049b48e4cf0ea863e8f08d6e7a
